default.nix

```diff
@@ -1,2 +1,20 @@
 {nixpkgs ? import <nixpkgs> {}, compiler ? "ghc865"}:
-nixpkgs.pkgs.haskell.packages.${compiler}.callPackage (import ./src/default.nix) {}
+let
+  dapptools = builtins.fetchGit {
```

Comment:
Why is this needed?

Reply:
The nix packages for `hevm` and `sbv` are broken in `nixpkgs` so we need to manually override them right now

Reply:
Ok!